game/src/game/player.rs

```diff
@@ -29,26 +60,66 @@ pub struct PlayerGfxBundle {
     sprite: SpriteSheetBundle,
     animation: SpriteAnimationBundle,
 }
-
+//where is this added/spawned, currently debug spawned only
 #[derive(Component, Default)]
 pub struct PlayerBlueprint;
 
-#[derive(Component, Default)]
-pub struct PlayerGent;
+#[derive(Component)]
+pub struct PlayerGent {
+    e_gfx: Entity,
```

Comment:
Good. This was supposed to be there to begin with. It is good to keep bidirectional links between Gfx and Gent.